platforms/corda-ent/configuration/roles/setup/tlscerts/tasks/main.yaml

```diff
@@ -0,0 +1,39 @@
+# This role pulls the idman & networkmap certs from the vault of CENM and pushes it to the vault of all other organizations
+
+# Check if the idman tls certificate is already present in the vault or not
+- name: Check if the idman tls certificate is already present in the vault or not
+  shell: |
+    vault read -field=idman.crt secret/{{ org.name }}/certs/"{{ network | json_query('orderers[?type==`idman`].name') | first }}"
```

Comment:
use the json_query as an input rather than repeating the function call multiple times